tests/e2e/util.go

```diff
@@ -4779,11 +4796,12 @@ func scaleDownStatefulSetPod(ctx context.Context, client clientset.Interface,
 scaleUpStatefulSetPod is a utility method which is used to scale up the count of StatefulSet replicas.
 */
 func scaleUpStatefulSetPod(ctx context.Context, client clientset.Interface,
-	statefulset *appsv1.StatefulSet, namespace string, replicas int32, parallelStatefulSetCreation bool) {
+	statefulset *appsv1.StatefulSet, namespace string, replicas int32,
+	parallelStatefulSetCreation bool, is256Disk bool) {
```

Comment:
instead of `is256Disk`, you can use `useLargeTimeout`
also you can make this as a variadic argument, with which we do not have to pass this from all the calls

Reply:
Ack